Source/Dafny/Compilers/Compiler-Csharp.cs

```diff
@@ -1636,13 +1701,21 @@ protected class ClassWriter : IClassWriter {
         }
         cw.StaticMemberWriter.WriteLine(";");
       } else if (cw.CtorBodyWriter == null) {
-        cw.InstanceMemberWriter.Write($"{publik}{konst} {typeName} {name}");
+        if (isPublic && !isConst) {
```

Comment:
Why is the `!isConst` check here?

Reply:
Thank you for catching this! I don't think this check should be made here.